extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/AwsApacheHttpClientConfig.java

```diff
@@ -0,0 +1,119 @@
+package io.quarkus.dynamodb.runtime;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class AwsApacheHttpClientConfig {
+
+    /**
+     * Connection Acquisition Timeout
```

Comment:
We usually don't all capitalize the javadoc. It should be `Connection acquisition timeout`.

Granted it will be very tedious to change that so not a blocker for me.

Reply:
I de-captilized it anyway :)